modules/client/src/main/java/org/apache/ignite/internal/client/TcpClientChannel.java

```diff
@@ -159,17 +154,6 @@ private TcpClientChannel(
 
         log = ClientUtils.logger(cfg.clientConfiguration(), TcpClientChannel.class);
 
-        this.timeoutWorker = new TimeoutWorker(
-                log,
-                cfg.getAddress().getHostString() + cfg.getAddress().getPort(),
-                "TcpClientChannel-timeout-worker",
-                pendingReqs,
-                // Client-facing future will fail with a timeout, but internal ClientRequestFuture will stay in the map -
-                // otherwise we'll fail with "protocol breakdown" error when a late response arrives from the server.
-                false,
```

Comment:
May be you will remove this flag from the common timeout worker because it was written for the client and now it is useless.

Reply:
Fixed.